### DISCUSSION TITLE
How to revert the icon change for the menu button and newtab page?

### DISCUSSION THREAD

**Discussion by ItzELECTR0** [Godiesc/firefox-gx#218] [Q&A] [answered]
The logo icon in the menu button has changed to this thing
![image](https://github.com/user-attachments/assets/9bf51940-fa13-4451-8656-08d983716ceb)
Same thing with newtab
![image](https://github.com/user-attachments/assets/d7cd8621-0f68-4d1e-a9bf-3830f6c5b5bb)
I'm not really a fan of it, is there a way to revert to the old one? It's still front and center in the readme and can be found in the icons directory
![image](https://github.com/user-attachments/assets/6d86f8f0-9784-41e5-b8ed-2229f00e5379)

**Comment by Godiesc** [comment, marked as answer]
hi,
you can make this change with 2 steps:

1. Paste the previous logo (`foxmenu.svg`) image into `chrome/icons` folder:
<img src="https://github.com/user-attachments/assets/bd0e0b70-3b0f-4f4d-b781-5db63e1a8b94" width="50" height="50 "/>

2. Copy and paste this code at the end of the `chrome/components/ogx_UC-newtabpage.css` file:

```
.alt-stop { stop-color: var(--fuchsia) !important; }                                          /* left color logo-and-wordmark */
.main-stop { stop-color: var(--blue) !important; }                                           /* right color logo-and wordmark*/

@-moz-document url(chrome://browser/content/browser.xul), url(about:newtab), url(about:home), url(about:privatebrowsing) {
    .logo{
        background: url("../icons/foxlogo.svg") no-repeat center/90px !important;
        width: 90px !important;
        height: 92px !important;
    }
    .wordmark {
        background: url("../icons/firefox-wordmark.svg") no-repeat center/140px !important;
        width: 140px !important;
    }
}
```
